parsl/launchers/launchers.py

```diff
@@ -373,6 +373,50 @@ def __call__(self, command, tasks_per_node, nodes_per_block, walltime=None):
         return x
 
 
+class JsrunLauncher(Launcher):
+    """  Worker launcher that wraps the user's command with the Jsrun launch framework
+    to launch multiple cmd invocations in parallel on a single job allocation
+
+    """
+    def __init__(self, overrides=''):
+        """
+        Parameters
+        ----------
+
+        overrides: str
+             This string will be passed to the aprun launcher. Default: ''
+        """
+        self.overrides = overrides
+
+    def __call__(self, command, tasks_per_node, nodes_per_block, walltime=None):
```

Comment:
this line should be updated to remove walltime, as in PR #1177. This code should still work without removing that, but it's messy.

Reply:
Done